test/Microsoft.VisualStudio.Editor.Razor.Test/BraceSmartIndenterIntegrationTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Test;
+using Microsoft.VisualStudio.Text;
+using Xunit;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    public partial class BraceSmartIndenterIntegrationTest : BraceSmartIndenterTestBase
```

Comment:
Which is it? are they using inheritance or partials?

Reply:
lmao, well lets make this a `public partial abstract internal protected facepalm class`